src/components/chips/js/chipsController.js

```diff
@@ -183,7 +183,7 @@ MdChipsCtrl.prototype.getPlaceholder = function() {
   // Allow `secondary-placeholder` to be blank.
   var useSecondary = (this.items.length &&
       (this.secondaryPlaceholder == '' || this.secondaryPlaceholder));
-  return useSecondary ? this.placeholder : this.secondaryPlaceholder;
+  return useSecondary ? this.secondaryPlaceholder : this.placeholder;
```

Comment:
Nope, our existing logic has been wrong the whole time. It currently says

    if use secondary
        use regular placeholder
    else
        use secondary placeholder

which is totally backwards. This fixes it. See #2770 for Tyler's comment about it being backwards.